packages/teleterm/src/ui/DocumentCluster/ClusterResources/Servers/useServers.ts

```diff
@@ -18,6 +18,19 @@ import { useAppContext } from 'teleterm/ui/appContextProvider';
 
 import { useServerSideResources } from '../useServerSideResources';
 
+function getEmptyTableText(status: '' | 'error' | 'processing' | 'success') {
```

Comment:
I'd probably reexport `status` from `Attempt` from `useAsync` as `AttemptStatus` instead of copying it over. If we ever refactor `useAsync`, it'll be more clear which places need changing (or even TypeScript will be able to automatically substitute some strings for us).